### DISCUSSION TITLE
Q: Clickable Links and/or View Mode

### DISCUSSION THREAD

**Discussion by bryce-cc-dunn** [sibiraj-s/ngx-editor#460] [Q&A] [answered]
Hi, working with ngx-editor and I seem to have the toolbar and inserting a link part working fine. It ends up showing as you'd expect a link would.

![image](https://user-images.githubusercontent.com/81773820/228974084-c0a35b2f-939b-490c-8461-3ecfd16d1558.png)

But I'm unable to click on the link to open it (left/right click, double click, nothing works).

I've tried marking the editor as disabled (to enter a sort of "view mode") but this still renders the links as unclickable.

When I hover, it shows the link address in a tooltip as I'd expect (see below) and likewise I can right click and hit copy link address and it copies the expected URL. But I'd expect to right click on the link and it automatically open.

![image](https://user-images.githubusercontent.com/81773820/228974273-df0eeebb-d9da-4ad1-8b93-3b46f2ba8028.png)

Am I missing something? Is there a view mode that is different from marking editor as disabled that activated the links to be clickable? Or is this simply not possible with the current editor (which would surprise me given the 'link' button in the toolbar)?

**Comment by sreerampendyala** [comment]
Facing similar issue.

**Comment by sibiraj-s** [comment, marked as answer]
![Screenshot 2023-04-11 at 12 46 33 PM](https://user-images.githubusercontent.com/20282546/231084887-0e09a9e2-46dc-46ca-b511-ded29dac99d5.png)

I am able to right click to navigate the link 🤔 is this not working? 

What is the OS you were testing this on?

**Reply by bryce-cc-dunn** [reply]
I guess it was being used improperly on our end, and we didn't realize. Links are working now as expected, thanks

[a comment by an automated account is not shown]
